src/memgraph.cpp

```diff
@@ -160,7 +160,7 @@ DEFINE_VALIDATED_int32(monitoring_port, 7444,
                        "Port on which the websocket server for Memgraph monitoring should listen.",
                        FLAG_IN_RANGE(0, std::numeric_limits<uint16_t>::max()));
 // NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
-DEFINE_VALIDATED_int32(metrics_port, 9092, "Port on which the Memgraph server for exposing metrics should listen.",
+DEFINE_VALIDATED_int32(metrics_port, 9091, "Port on which the Memgraph server for exposing metrics should listen.",
```

Comment:
What was reason for changing this port?

Reply:
This is the default port for Kafka usually. Prometheus is 9090, so I decided going with 9091